### PR TITLE
Add support for TYPO3 v10.0

### PR DESCRIPTION
**Pull request by bnf**

### CODE REVIEW

**Comment by bnf** [comment]
Sorry, I didn't see #118 before, therefore this patchset has some duplications – still, I think this approach is a bit cleaner:

- uses class aliases for the testbase ViewHelper
- proper site configuration handling for frontend requests
- fixes site configuration to be read from (and caches to be written to) the temporary test instance directory, instead of the original root
- Does not promise ^10.0 compatibility, just ~10.0.0

**Comment by bnf** [comment]
Rebased on master to incorporate the recent removal of 7.6 support in https://github.com/Nimut/testing-framework/commit/a0737b49e32640022f8d004487df5ae0d69f4b28.

File: compat/v95/Bootstrap/Bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Nimut\TestingFramework\v87\Bootstrap;
```
Comment (simonschaufi) [inline compat/v95/Bootstrap/Bootstrap.php:2]: wrong namespace, or?

**Review by simonschaufi — CHANGES_REQUESTED** [review]

**Comment by IchHabRecht** [comment]
Resolved with #123

**Pull request closed without merging**